ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/overlay_routing_protocol_address_family.schema.yml

```
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  overlay_routing_protocol_address_family:
+    documentation_options:
+      filename: Fabric Variables
+      table: IPv6 Underlay Settings
+    type: str
+    description: |
+      Enable overlay EVPN peering with IPv6 addresses.
```

I think we should reword with:
```suggestion
      When set to `ipv6`, enable overlay EVPN peering with IPv6 addresses.
```